src/LBPair.sol

```diff
@@ -117,6 +116,28 @@ contract LBPair is LBToken, ReentrancyGuardUpgradeable, Clone, ILBPair {
             protocolShare,
             maxVolatilityAccumulator
         );
+
+        string memory tokenXSymbol = IERC20Metadata(address(_tokenX())).symbol();
+        string memory tokenYSymbol = IERC20Metadata(address(_tokenY())).symbol();
+
+        _tokenName = string.concat("Liquidity Book Token ", tokenXSymbol, " - ", tokenYSymbol);
+        _tokenSymbol = string.concat("LBT_", tokenXSymbol, "-", tokenYSymbol);
```

Comment:
the tokenName and Symbol are identified by the compound key(tokenX, tokenY, `binStep`).   
not including the `binStep` in the token symbol and name will be confusing if same token pairs have differen pools with differnet binStep respectively. 
pls add the bin step in the token symbol and name.  

import the library libraries/StringUtils and 
```suggestion
        string memory tokenXSymbol = IERC20Metadata(address(_tokenX())).symbol();
        string memory tokenYSymbol = IERC20Metadata(address(_tokenY())).symbol();
        string memory binStep = StringUtils.uint16ToString(_binStep());

        _tokenName = string.concat("Liquidity Book Token ", tokenXSymbol, "/", tokenYSymbol, "/", binStep);
        _tokenSymbol = string.concat("LBT_", tokenXSymbol, "/", tokenYSymbol, "/", binStep);
```